scripts/commands/checkMetadata.ts

```diff
@@ -76,18 +76,42 @@ const main = async (): Promise<void> => {
       Invalid markdown file metadata. Every .md and .mdx file should start with a metadata block like this:
 
       ---
-      title: OpenQASM 3 feature table
-      description: A list of the OpenQASM 3 language features
+      title: Representing quantum computers
+      description: Learn about coupling maps, basis gates and backend errors for transpiling
```

Comment:
```suggestion
      description: Learn about coupling maps, basis gates, and backend errors for transpiling
```
(teeny tweak: Oxford comma :))